### PR TITLE
Test pep8

### PR DESCRIPTION
**Pull request by pfertyk**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
-
+def hi():
+    x = a
```
Comment (pepophile) [inline main.py]: F821 undefined name 'a'

File: main.py [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+    x = 1
```
Comment (pepophile) [inline main.py]: F841 local variable 'x' is assigned to but never used

File: main.py [hunk 3]
```diff
@@ -1,3 +1,6 @@
...
+    y=3
```
Comment (pepophile) [inline main.py]: F841 local variable 'y' is assigned to but never used

Comment (pepophile) [inline main.py]: E225 missing whitespace around operator

File: .lintrc [hunk 4]
```diff
@@ -0,0 +1,2 @@
+[tools]
```
Comment (pfertyk) [inline .lintrc]: I would like to comment here.

File: main.py [hunk 5]
```diff
@@ -1,3 +1,6 @@
-
-if __name__=='__main__':
+def oh():
+    pass
+def hi():
```
Comment (pepophile) [inline main.py]: E302 expected 2 blank lines, found 0

**Pull request closed without merging**